templates/_worker.js/routes-matcher.ts

```diff
@@ -57,21 +59,21 @@ export class RoutesMatcher {
 		private output: VercelBuildOutput,
 		/** Request Context object for the request to match */
 		private reqCtx: RequestContext,
-		prevMatch?: MatchedSet
+		wildcardConfig?: VercelWildcardConfig
 	) {
 		this.url = new URL(reqCtx.request.url);
 		this.cookies = parse(reqCtx.request.headers.get('cookie') || '');
 
-		this.path = prevMatch?.path || this.url.pathname || '/';
-		this.status = prevMatch?.status;
-		this.headers = prevMatch?.headers || {
-			normal: new Headers(),
-			important: new Headers(),
-		};
-		this.searchParams = prevMatch?.searchParams || new URLSearchParams();
```

Comment:
was this not necessary? 🤔 😕

Reply:
I think the `prevMatch` was just left over from the initial version before it was converted to a class and has never been used since. No point in keeping it around.

Reply:
ah! we we're not even passing it to the function??

Reply:
ah yeah we're not 😮👍